### PR TITLE
Provide `SkyAppParamsConfig`

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**

### CODE REVIEW

File: lib/sky-pages-module-generator.js [hunk 1]
```diff
@@ -244,6 +249,12 @@ export function skyViewkeeperHostOptionsFactory(config: SkyAppConfig): SkyViewke
   return hostOptions;
 }
 
+export function paramsConfigFactory(config: SkyAppConfig): SkyAppParamsConfig {
```
Comment (Blackbaud-PaulCrowder) [inline lib/sky-pages-module-generator.js:252]: Should we call this `skyAppParamsConfigFactory` since we're exporting it?

**Review by Blackbaud-PaulCrowder — COMMENTED** [review]
Should we also provide `SkyAppRuntimeConfigParamsProvider` so consumers could switch over to using that to get param values?

File: lib/sky-pages-module-generator.js (hunk 1 above)
Comment (Blackbaud-SteveBrush) [reply lib/sky-pages-module-generator.js:252]: Done. Regarding your top comment, the `SkyAppRuntimeConfigParamsProvider` is provided in the `'root'`, so consumers can pull it into their components/services without a module providing it (or Builder, in this case).
https://github.com/blackbaud/skyux-config/blob/master/src/app/public/params-provider.ts#L17

Comment (Blackbaud-PaulCrowder) [reply lib/sky-pages-module-generator.js]: Got it.

**Review by Blackbaud-PaulCrowder — APPROVED** [review]

**Pull request merged**